Create ui-c8y-1019-6-3-3-d-rotation-model-shows-the-actual-model-5566.md (#1728)
SoftwareAG/c8y-docs: content/change-logs/ui-c8y-1019-6-3-3-d-rotation-model-shows-the-actual-model-5566.md

---
date: ""

> title: 3D rotation model shows the actual model (#5566)

Reviewer: ```suggestion
title: Rotation widget shows the actual model (#5566)
```
The PR number in brackets should not show up in the title, should it?

> product_area: undefined

Reviewer 2: ```suggestion
product_area: Application enablement \u0026 solutions
```

> product_area: Application enablement \u0026 solutions

Reviewer: ```suggestion
product_area: Application enablement \u0026 solutions
```
Will this cause any problems? \u0026 solutions

Reviewer 2: Hope not. @carlosceia do you think this might be an issue?

change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-57985
version: 1019.6.3
---

> An issue has been addressed that caused the 3D rotation widget to not show the actual model.

Reviewer: ```suggestion
A bug has been fixed that prevented the "Rotation" widget from displaying the actual model.
```
